mitodl/ocw-to-hugo
# don't process unpublished courses

#### Pre-Flight checklist


#### What are the relevant tickets?
Related to https://github.com/mitodl/hugo-course-publisher/issues/436
Closes https://github.com/mitodl/ocw-to-hugo/issues/149

#### What's this PR do?
This PR makes sure a course has not been unpublished before generating markdown and the data template

#### How should this be manually tested?
Run `ocw-to-hugo` with an unpublished course (`8-01sc-classical-mechanics-fall-2016` for example) and ensure that processed markdown and a data template do not appear in your output directory.
